### ISSUE TITLE
Payment Method Issue in Admin

### ISSUE THREAD

**Issue by cbend** [magento/magento2#11380]
### Preconditions
1. Magento EE 2.2.0

### Steps to reproduce
1. Only enable one payment method that has a form (PayPal, Purchase Order, etc.)
2. Place an order in the admin

### Expected result
1. The form to enter information for the payment method should display

### Actual result
1. The form does not display

![image](https://user-images.githubusercontent.com/25639152/31458886-5e7886c6-ae8f-11e7-9628-f23cf1d42a19.png)

### Note
1. This works fine on the frontend
2. It works fine if multiple payment methods are enabled

**Comment by magento-engcom-team** [comment]
@cbend, thank you for your report.
We've created internal ticket(s) MAGETWO-81797 to track progress on the issue.

**Comment by magento-team** [comment]
Internal ticket to track issue progress: MAGETWO-82653

**Comment by pniel-cohen** [comment]
@magento-team, Is there any fix/patch/workaround for this issue?
I'm having the same on Magento 2.2.2 with Authorize.net

**Comment by baublet** [comment]
Ping. This is kind of a big deal to fix.

**Comment by baublet** [comment]
Temporary fix:

```
// file: app/code/Namespace/Module/view/adminhtml/web/js/showSoloPaymentOption.js

// Temporary fix for https://github.com/magento/magento2/issues/11380

setInterval(function() {
    var list = document.querySelectorAll(
        "#order-billing_method_form .admin__field-option"
    )
    if (list.length !== 1) return
    // Find the payment method field and show it
    var panel = document.querySelector(
        "#order-billing_method_form .admin__page-section-item"
    )
    if (!panel || panel.style.display !== "none") return
    // If we get here, we have a single payment method, a valid corresponding
    // panel, and it's hidden, so let's show it.
    panel.style.display = "block"
    // Check this every 1.5 seconds
}, 1500)
```

```
<!-- file: app/code/Namespace/Module/view/adminhtml/layout/sales_order_create_index.xml -->
<?xml version="1.0"?>
<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" layout="admin-1column" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
    <head>
        <script src="Namespace_Module::js/showSoloPaymentOption.js"/>
    </head>
</page>
```

**Comment by magento-engcom-team** [comment]
Hi @baublet
This issue already fixed in private repository and will be merged very soon

**Comment by baublet** [comment]
Excellent, thanks!

**Comment by vnaz** [comment]
The same problem on Magento v 2.2.3
Is it fixed on 2.2.4 ?

**Comment by ghost** [comment]
Hi @cbend thank you for you report, the issue was tested and we can confirm that it's already fixed in 2.3-develop branch and available on 2.3.1 release -> https://github.com/magento/magento2/pull/11683
